pom.xml

```diff
@@ -272,6 +267,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.0</version>
```

Comment:
it seem to work without overriding the plugin versions also (I tried building locally). I think we should remove them, since we don't have specific requirement. They will be upgraded along with Spring BOM, whenever we upgrade Spring.